packages/api/src/index.ts

```diff
@@ -50,9 +50,13 @@ export type AppState = {
   erc20s: uma.tables.erc20s.JsMap;
   stats: {
     usd: {
-      latest: empStats.JsMap;
+      latest: {
+        tvl: empStats.JsMap;
+        tvm: empStats.JsMap;
+      };
       history: {
-        [address: string]: empStatsHistory.SortedJsMap;
+        tvl: empStatsHistory.SortedJsMap;
+        tvm: empStatsHistory.SortedJsMap;
```

Comment:
tvl and tvm are given their own tables now. these tables can store data for multiple emps.